www/static/js/algolia.js

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+{% if page.version %}
+    {% assign version = page.version %}
+{% else %}
+    {% assign version = site.latest_docs_version %}
+{% endif %}
+
+{% if page.language %}
+    {% assign language = page.language %}
+{% else %}
+    {% assign language = site.language %}
+{% endif %}
+
+docsearch({
+        apiKey: '{{ site.algolia_api_key }}',
+        indexName: 'cordova',
+        inputSelector: '#header-search-field',
+        algoliaOptions: { 'facetFilters': ["version:  {{ version }}", "language: {{ language }}"] }
```

Comment:
Two notes:
- in the example from the Algolia folks, I think there was no space between `version:` and `{{version}}`; same for the language
- the two variables need to be generated as constants on the HTML page (maybe bring back `algolia.html` include) and then used in this file